connector/google/google.go

```diff
@@ -23,8 +23,8 @@ type Config struct {
 	Scopes []string `json:"scopes"`
 	// Optional list of whitelisted domains when using Google
 	// If this field is nonempty, only users from a listed domain will be allowed to log in
-	HostedDomains []string `json:"hostedDomains"`
-	filePath  string `json:"serviceAccountCred"`
+	HostedDomain string `json:"hostedDomain"`
+	FilePath  string `json:"serviceAccountCredPath"`
```

Comment:
let's have the name of the field on the struct match the json name for readability and consistency